src/ring_theory/polynomial/pochhammer.lean

```diff
@@ -92,6 +92,11 @@ begin
     refl, },
 end
 
+lemma pochhammer_succ_eval (n : ℕ) (k : S) :
+  (pochhammer S n.succ).eval k = (pochhammer S n).eval k * (k + ↑n) :=
```

Comment:
Could we state this using `(n+1)` instead of `n.succ`?

Reply:
Done!

I'll wait until this passes CI and merge, thanks!